### ISSUE TITLE
wish:  to add more than one icon

### ISSUE THREAD

**Issue by AndyLicht** [jonmiles/bootstrap-treeview#99]
It would be a nice feature to add more than one icon. and it would be very nice to change from left to right

**Comment by biomcanx** [comment]
I agree I would like to be able to add more than 1 icon.  I use icons for links to other things...details edit delete etc.
